extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -205,8 +204,9 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 				width: this.componentMaxLength + 20,
 				justifyContent: 'flex-start'
 			}).component();
-			const warningButton = this.createWarningButton(constants.outputColumnDataTypeNotSupportedWarning);
+			const warningButton = this.createWarningButton();
 			warningButton.onDidClick(() => {
+				this.openWarningCalloutDialog(constants.columnDataTypeMismatchWarningHeading, 'output table row dialog', constants.outputColumnDataTypeNotSupportedWarning, constants.learnMoreLink, constants.mlExtDocLink);
```

Comment:
the second parameter is for telemetry, please do not use spaces for names and the name is not unique or clear enough. can you come up with another name? this also applies to the other place where openwarningClloutDialog is called

Reply:
Corrected